bin/test-entrypoint.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 export PATH=$PATH:~/.local/bin
 
-
 set -e
```

Comment:
And does anyone remember what this is for?

Reply:
I would like to know this as well. Is it necessary or can we remove it?

Reply:
Should we leave it in or take it out?

Reply:
OK, if none of us remember its use, then it's probably trivial or unimportant in the grand scheme - and is definitely a deviation from the 2017 entrypoint.

Please remove it @nam20485

Reply:
Removed in latest commit.